example/integration_test/scenario_device_id_init/1.1.BV_dev_type_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+
+import 'package:countly_flutter/countly_flutter.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import '../utils.dart';
+
+/// 1.Check if setDeviceId() sets the device ID correctly and the type is DEVELOPER_SUPPLIED
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  testWidgets('1.1.Test Device ID Type - BV - Developer sets the ID during init', (WidgetTester tester) async {
+    // Initialize the SDK
+    CountlyConfig config = CountlyConfig(SERVER_URL, APP_KEY).setLoggingEnabled(true).setDeviceId('');
+    await Countly.initWithConfig(config);
+    // Get the device ID type
+    DeviceIdType? type = await Countly.getDeviceIDType();
+    DeviceIdType? newModuleType = await Countly.instance.deviceId.getIDType();
+    String? id = await Countly.getCurrentDeviceId();
```

Comment:
is this test aims to check old calls

we have now Countly.instance.deviceId.getID() method right?